src/components/tables/hovered_table_store.ts

```diff
@@ -48,9 +44,25 @@ export class HoveredTableStore extends SpreadsheetStore {
     if (!table) {
       return;
     }
-    const { left, right } = table.range.zone;
+    const { left, right, top } = table.range.zone;
+    const isTableHeader = row < top + table.config.numberOfHeaders;
+    const isTableRowHasContent = (): boolean => {
```

Comment:
```suggestion
    const doesTableRowHaveContent = (): boolean => {
```
nitpick. Or change to `isTableRowEmpty`